guides/source/asset_pipeline.md

```diff
@@ -45,7 +49,7 @@ gem 'coffee-rails'
 ```
 
 Using the `--skip-sprockets` option will prevent Rails 4 from adding
-`sass-rails` and `uglifier` to your Gemfile, so if you later want to enable
+these gems to your Gemfile, so if you later want to enable
```

Comment:
#23479 removes all 3 gems, so if you want to re-enable sprockets you should add all of them back, no?